elyra/templates/pipeline/pipeline_properties_template.jinja2

```diff
@@ -0,0 +1,116 @@
+{
+  "current_parameters": {
+    "name": "",
+    "runtime": "",
+    "description": "",
+    "elyra_runtime_image": null,
+    "elyra_env_vars": []
+  },
+  "parameters": [
+    {
+      "id": "name"
+    },
+    {
+      "id": "runtime"
+    },
+    {
+      "id": "description"
+    },
+    {
+      "id": "elyra_runtime_image"
+    },
+    {
+      "id": "elyra_env_vars"
+    }
+  ],
+  "uihints": {
+    "id": "nodeProperties",
+    "parameter_info": [
+      {
+        "control": "custom",
+        "custom_control_id": "DisplayControl",
+        "parameter_ref": "name",
+        "label": { "default": "Pipeline Name" }
+      },
+      {
+        "control": "custom",
+        "custom_control_id": "DisplayControl",
+        "parameter_ref": "runtime",
+        "label": { "default": "Pipeline Runtime" }
+      },
+      {
+        "control": "custom",
+        "custom_control_id": "StringControl",
+        "parameter_ref": "description",
+        "label": { "default": "Pipeline Description" },
+        "data": {
+          "placeholder": "Pipeline description",
+          "format": "multiline"
+        }
+      },
+      {
+        "parameter_ref": "elyra_runtime_image",
+        "control": "custom",
+        "custom_control_id": "EnumControl",
+        "label": { "default": "Runtime Image" },
+        "description": {
+          "default": "Docker image used as execution environment.",
```

Comment:
Can we change this here to 'Container` image (and do the same for the help text of the local property)? This aligns us better with the terminology we are already using in most of the documenation.

```suggestion
          "default": "Container image used as execution environment.",
```

Reply:
I'm ok with changing this text (as long as we change it everywhere). I'll do it now

Reply:
done